docs/index.md

```diff
@@ -460,6 +460,35 @@ async def app(scope, receive, send):
     })
 ```
 
+## The WSGI Support
+
+Uvicorn offer support to WSGI applications throught `WSGIMiddleware` which converts
+your WSGI into ASGI application seamlessly just by running it with `--interface wsgi`
```

Comment:
Is this the first mention to the `WSGIMiddleware` on the documentation? I think this is just an implementation detail.

Reply:
Also cut it and put just the mention in the settings like you said